### DISCUSSION TITLE
P-BACKGROUND

### DISCUSSION THREAD

**Discussion by eairafao** [po-ui/po-angular#1137] [Tire suas dúvidas] [answered]
#### Descrição da melhoria
Olá, não é possível mudar o p-background para o lado esquerdo? dentro de po-page-login

Outra dúvida, não é possível mudar a cor do background? todos os projetos que observei da comunidade possui o mesmo fundo que o meu (branco/gelo)

#### Caso de uso
Criatividade em diferentes projetos

**Comment by alinelariguet** [comment, marked as answer]
Bom dia @eairafao 
É possível [customizar o nosso tema padrão](https://po-ui.io/guides/colors-customization) e também é possível [criar um tema customizado](https://po-ui.io/guides/create-theme-customization).
Att.

**Reply by eairafao** [reply]
Obrigado pelo auxilio!
